site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -45,24 +45,77 @@ postman files later, you can just create a new environment again to work with
 Select the workspace menu at the top of the Postman interface, and create a new one
 called 'Egeria dojo':
 
-![Creating a new Environment 1](postman-workspace1.png)
-![Creating a new Environment 2](postman-workspace2.png)
-![Creating a new Environment 3](postman-workspace3.png)
+![Creating a new Environment 1](postman-create-workspace1.png)
+![Creating a new Environment 2](postman-create-workspace2.png)
+![Creating a new Environment 3](postman-workspace-created.png)
 
 ## Importing Egeria Postman data
 
 We now will
-* Select 'Import' and select 'GitHub' as the source
-* Authenticate as required
-* Select the "main" branch from the odpi/egeria repository on GitHub
-* Import all the Postman data found
-
-![Import Begin](postman-import-begin.png)
-![Select import source](postman-import-coderepo.png)
-![Auth with GitHub](postman-import-gotogh.png)
-![Select egeria repo](postman-import-ghrepo.png)
-![Select what to import](postman-import-select.png)
-![After Import](postman-import-after.png)
+
+* Individually import the Egeria Postman collections required for this dojo
+* Select 'Import' and select 'Link' as the source
+* Paste the URL found below for the `Egeria-admin-services-operational` collection
+
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/admin-services/Egeria-admin-services-operational.postman_collection.json
```

Comment:
I'm thinking how we make this easier. One option is to change these links into something like:
<img width="1209" alt="Screenshot 2023-02-28 at 09 34 59" src="https://user-images.githubusercontent.com/7292002/221812383-c6d3a939-32fb-403e-a28c-7fbe33944032.png">

This will enable a 'copy' link 

Unfortunately the button overlaps with the text, and the .url filetype here is just made up. I will look to see if there's a better way.

<img width="794" alt="Screenshot 2023-02-28 at 09 34 15" src="https://user-images.githubusercontent.com/7292002/221812400-8647ddcf-6170-440d-8681-f89ed5faa83c.png">

This is useful though, since a triple click on the URL will result in
<img width="779" alt="Screenshot 2023-02-28 at 09 37 53" src="https://user-images.githubusercontent.com/7292002/221813304-30eed26b-1fd7-4a0a-b1bf-4dde5391fe30.png">

Reply:
I tried to find other approaches, but your suggestion seemed to be the best. I used this through the document when listing a URL.